development/cpp/creating_android_modules.rst

```diff
@@ -216,6 +216,21 @@ add something like this:
 Now you can refer to those resources by their id (``R.string.my_string``, and the like)
 by importing the ``com.godot.game.R`` class in your Java code.
 
+Assets
+---------
```

Comment:
The underline should have the same amount of (fixed size) character as the title, otherwise Sphinx will generate a warning (yeah, it's a bit annoying :)).